spec/models/poll/question_spec.rb

```diff
@@ -46,4 +46,30 @@
       end
     end
   end
+
+  describe "#options_total_votes" do
```

Comment:
Since we're at it, shall we add a test that checks that options for other questions aren't included in the count, even if they've got the same `answer:` value?